packages/protocol/contracts-0.8/common/GasPriceMinimum.sol

```diff
@@ -175,7 +176,7 @@ contract GasPriceMinimum is
    * For other addresses it returns gasPriceMinimum() mutiplied by 
    * the SortedOracles median of the token. It does not check tokenAddress is a valid fee currency.
    * this function will never returns values less than ABSOLUTE_MINIMAL_GAS_PRICE.
-   * If Oracle rate doesn't exist, it returns ABSOLUTE_MINIMAL_GAS_PRICE.
+   * If Oracle rate doesn't exist, it reverts.
```

Comment:
There's a question of what's the desired behavior. Sounds like in the past we *intended* to return `ABSOLUTE_MINIMAL_GAS_PRICE` (defined to be 1 wei).

1. If that is the desired behavior, the fix should be instead to return 0 from the function above, instead of reverting.
2. But likely it's a good idea to not return such a tiny gas price, in which case sounds like we're good to remove any references to `ABSOLUTE_GAS_PRICE_MINIMUM`, including the `max` calculation below.

I guess the one thing to check is how would the celo-blockchain client react to a revert here?

Reply:
I think reverting is ok.

But sure, need to check what happens on the client

Reply:
The problem is that it can technically be 0 even for whitelisted tokens that do not use adapter. This is not yet supported by client but in long run it should be. Then it is up to user whether they will choose to pay with this token even though it is more expensive.

Personally I would keep the revert.

Reply:
I agree we should revert (like we've been doing, now it'd be more explicit).
@marekolszewski The comment it's a typo, should have been how Pavel left it now.